src/routes/api_docs.ts

```diff
@@ -0,0 +1,26 @@
+import type { Router } from "express";
+import swaggerUi from "swagger-ui-express";
+import { readFile } from "fs/promises";
+import { fileURLToPath } from "url";
+import { dirname, join } from "path";
+import yaml from "js-yaml";
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+const swaggerDocument = yaml.load(
+    await readFile(join(__dirname, "../etapi/etapi.openapi.yaml"), "utf8")
+) as object;
```

Comment:
bit of a nitpicking comment:

the swaggerUI types seem to suggest that it takes a `JsonObject` type
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/84e6da3397915d415adf7ef5b636affd8638b447/types/swagger-ui-express/index.d.ts#L38

the js-yaml types suggest that "load" returns an `unknown` type
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/84e6da3397915d415adf7ef5b636affd8638b447/types/js-yaml/index.d.ts#L3

So I think if we do a typecast here, it should be OK to typecast this to "JsonObject" instead.

what do you think?

(you might need to use `import type` to import the definition)